examples/long_running_measurement/poetry.toml

```diff
@@ -0,0 +1,2 @@
+[virtualenvs]
+in-project = true
```

Comment:
What does this file and this 'in-project' line do for us again?

Reply:
It tells Poetry to create a virtualenv in `.venv` rather than `C:\Users\username\AppData\Local\pypoetry\Cache\virtualenvs\long-running-measurement-asdfghjk-py3.9`.